grafana_client/elements/plugin.py

```diff
@@ -9,60 +11,83 @@ def __init__(self, client):
         self.client = client
         self.logger = logging.getLogger(__name__)
 
-    def get_installed_plugins(self):
+    def list(self):
         """
-        :return:
+        Return list of all installed plugins.
         """
         path = "/plugins?embedded=0"
         r = self.client.GET(path)
         return r
 
-    def install_plugin(self, pluginId, version, errors="raise"):
+    def by_id(self, plugin_id):
+        """
+        Return single plugin item selected by plugin identifier.
+        """
+        plugins = self.list()
+        return get_plugin_by_id(plugin_list=plugins, plugin_id=plugin_id)
+
+    def install(self, plugin_id, version=None, errors="raise"):
         """
-        : return:
+        Install a 3rd-party plugin from the plugin store.
         """
         try:
-            path = "/plugins/%s/install" % pluginId
-            r = self.client.POST(path, json={"version": version})
+            path = "/plugins/%s/install" % plugin_id
+            # Unfortunately, this endpoint may respond with an empty JSON,
+            # which needs compensation, because it does not decode well.
+            r = self.client.POST(path, json={"version": version}, accept_empty_json=True)
```

Comment:
Thanks for handling this , I was not sure on how to handle the exceptions I was getting.